### ISSUE TITLE
Wrong handling of 'undocumented' attributes with jsdoc 3.6.x

### ISSUE THREAD

**Issue by Alexis-ROYER** [englercj/tsd-jsdoc#99]
As noticed by [jsdoc#1672](https://github.com/jsdoc/jsdoc/issues/1672), jsdoc 3.6.x output differs from 3.5.x, especially on `undocumented` attributes strangely set to `true` while the `comment` attribute is correctly set with some user comment.

The effect for tsd-jsdoc is that such doclets are not managed anymore.

Steps to reproduce :
- checkout tsd-jsdoc, run `npm install` but then force the installation of jsdoc with version 3.6.3.
- `npm run build`
- `npm run test` => failure

**Comment by englercj** [comment]
Yeah there are a bunch of backwards incompatible changes to jsdoc 3.6. I really wish they had done a major release.

I think in this lib we need to update to work with 3.6, and change how we determine exports (a la #100), then do a major release on our side.

**Comment by Alexis-ROYER** [comment]
I've just committed a sample of code to fix this problem.
See changes on [publish.ts](https://github.com/englercj/tsd-jsdoc/pull/98/commits/82e01e4410c2187c46aceedc8c233b426c550f8e#diff-d9a60851031c31a6d1f30fd69b639407) in [PR#98](https://github.com/englercj/tsd-jsdoc/pull/98).
